handbook/people-ops/requesting-time-off.md

```diff
@@ -1 +1,11 @@
-# Requesting time off
\ No newline at end of file
+# Requesting time off
+
+The process outlined below is today's process, but please be aware that the People Ops team is actively working on implementing a new one that will provide a platform for requesting time off via HRIS. 
+
+Current expectations for requesting time off:
+
+- Notify your manager with as much notice as possible– notify your manager via the channel you regularly use to communicate other important information (e.g. email, Slack, 1-1)
+- Once approved by your manager, create a calendar event (on your own calendar) like the one outlined on this [doc](https://docs.google.com/document/d/1BIq-4fZmLHYiX7z-pLfeLqmLiP9z5cX9hy851H-D2Js/edit?usp=sharing)
```

Comment:
Would it be helpful for me to add this as a screenshot instead of linking out to a separate doc? I can do it quickly if so

Reply:
@dadlerj if it's an easy edit then that would be perfect!